packages/frontity-org-theme/src/components/styles/global-styles.tsx

```diff
@@ -114,6 +114,16 @@ const elementBase = (colors: FrontityOrg["state"]["theme"]["colors"]) => css`
   }
   ul {
     list-style-type: "→ ";
+
+    & > li {
+      margin: 1.1rem 0;
+      line-height: 1.3;
+
+      &::before {
+        line-height: 0;
+        margin-right: 0.5rem;
```

Comment:
If you change the line-height to `1.5rem` as suggested before, it's higher than the height of the icon, so you'll need to add `auto` margin to the top and the bottom to center it vertically. I think you need to change the `margin-right` to `margin: auto 0.5rem auto 0` right?

Reply:
Yeah, that works.